packages/demo/src/configuration/blocks/generators/fireworksBlock.fragment.glsl

```diff
@@ -29,14 +29,14 @@ vec4 mainImage(vec2 vUV) { // main
     vec2 origin = vec2(0.);
 	vUV.x *= aspectRatio;
     
-    for (float j = 0.; j < EXPLOSION_COUNT; ++j)
+    for (float j = 0.; j < fireworks; ++j)
     {
-        vec3 oh = hash31((j + 1234.1939) * 641.6974);
+        vec3 oh = hash31((j + 800.) * 641.6974);
```

Comment:
Seed value. I updated it so that the first few fireworks in our loop are more centered, in case someone sets the fireworkCount to something low.